rosco-core/src/test/groovy/com/netflix/spinnaker/rosco/providers/azure/AzureBakeHandlerSpec.groovy

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.rosco.providers.azure
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.netflix.spinnaker.rosco.api.Bake
+import com.netflix.spinnaker.rosco.api.BakeOptions
+import com.netflix.spinnaker.rosco.api.BakeRequest
+import com.netflix.spinnaker.rosco.providers.azure.config.RoscoAzureConfiguration
+import spock.lang.Shared
+import spock.lang.Specification
+import spock.lang.Subject
+
+class AzureBakeHandlerSpec extends Specification{
+
+  @Shared
+  String configDir = "/some/path"
+
+  @Shared
+  RoscoAzureConfiguration.AzureBakeryDefaults azureBakeryDefaults
+
+  void setupSpec() {
+    def azureBakeryDefaultsJson = [
+      templateFile: "azure-linux.json",
+      baseImages: [
+        [
+          baseImage: [
+            id: "ubuntu",
+            detailedDescription: "Ubuntu Server 14.04.4-LTS",
+            packageType: "DEB",
+          ]
+        ],
+        [
+          baseImage: [
+            id: "centos",
+            detailedDescription: "OpenLogic CentOS 7.1.20150731",
+            packageType: "DEB",
```

Comment:
This should probably be `RPM`, just in case anybody comes looking at our tests for example usage.